<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCDEncrdecr</class>
 <widget class="QDialog" name="RCDEncrdecr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Test</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRuns">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelRuns">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Runs:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper Index: (Max 256)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBenPar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set benchmarking parameters:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWarning_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LABEL</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLower">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lower Index: (Min. 8)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Memory Size (MB):</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinUpper">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>8</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutofill">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Autofill Sample Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chart Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiplication Factor: x2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLinear">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Linear Step:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLinear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Linear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Logarithmic (Base 2)</string>
    </property>
   </item>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
